homeassistant/components/tplink/entity.py

```diff
@@ -52,17 +52,14 @@
     DeviceType.Bulb,
     DeviceType.LightStrip,
     DeviceType.Dimmer,
+    DeviceType.Thermostat,
 }
 
 # Features excluded due to future platform additions
 EXCLUDED_FEATURES = {
-    # climate
-    "target_temperature",
-    "thermostat_mode",
     # update
     "current_firmware_version",
     "available_firmware_version",
-    "frost_protection_enabled",
```

Comment:
We should probably remove this feature altogether, as the thermostat on/off state is controlled by the same variable at the moment and I don't think there are any advantages for having a separate control for it.